proto/identity/login.proto

```diff
@@ -38,3 +57,11 @@ message LoginMessage {
   string unsignedMsg = 2;
   bytes signature = 3;
 }
+
+message UserLogin {
```

Comment:
Does the login need to provide all of this information? 

You are probably correct in that. Since you haven't had any implementation in this PR, it is hard for me to evaluate it.